src/main/scala/com/feedzai/cosytest/DockerComposeSetup.scala

```diff
@@ -298,12 +298,12 @@ case class DockerComposeSetup(
     * @param containerPath  Path to file destination in the container.
     * @return true if the file was copied successfully, otherwise false.
     */
-  def copyToHost(containerId: String, hostPath: String, containerPath: String): Boolean = {
+  def copyToHost(containerId: String, hostPath: Path, containerPath: Path): Boolean = {
```

Comment:
Not sure if arguments should be inverted. What do you think?

Reply:
Yes, for consistency I think they should. The javadoc is also inconsistent with the method.